app/router.options.js

```
@@ -0,0 +1,33 @@
+        if (savedPosition) {
+            return savedPosition
+        } else {
+            return new Promise(async (resolve) => {
+                nuxtApp.hooks.hookOnce('page:finish', async () => {
+                    await nextTick()
+                    resolve({
+                        top: 0,
+                        behavior: 'smooth'
+                    })
+                })
+            })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)